contrib/chibi-generate-install-meta-helper.cmake

```diff
@@ -0,0 +1,10 @@
+
+execute_process(
+    COMMAND find ${LIBDIR} -name "*.sld"
```

Comment:
Perhaps you meant capturing `find` result in another variable and feed it as argument to built chibi-scheme? You'll need two `execute_process`es then. It'd be nice if we can rewrite it with `file(GLOB_RECURSE)`.

Reply:
So this was an intentional choice because `tools/generate-install-meta.scm` reads from stdin and wanted to keep it simple; this way, the `find` output is piped through the following command, which works as far as I can tell. With `file(GLOB_RECURSE)`, we would need to write the glob result to disk first; then pass that file to the `execute_process`. This is more complicated, do we want that?

Reply:
Okay, leave it for now. I'll implement it for Win32 later.